go/pkg/events/events.go

```diff
@@ -112,7 +116,40 @@ func EventType(src []byte) (string, error) {
 	return string(name), nil
 }
 
+func VerifySignature(src []byte, publicKey any) error {
```

Comment:
you didn't change this, right?
i mean, it cannot work as you are not getting the `expire_at` value

Reply:
the golang part is wip